Changelog sep 5 (#4398)
timescale/docs: about/changelog.md

All the latest features and updates to $CLOUD_LONG.

## New data import wizard
<Label type="date">September 5, 2025</Label>

> We’ve introduced a cleaner, more intuitive UI for data import. It highlights the most common and recommended option—PostgreSQL Dump & Restore—while organizing all import options into clear categories, to make navigation easier.

Reviewer: ```suggestion
To make navigation easier, we’ve introduced a cleaner, more intuitive UI for data import. It highlights the most common and recommended option, Postgres Dump & Restore, while organizing all import options into clear categories, .
```

Author: partly implemented

The new categories include:

> - **PostgreSQL Dump & Restore**

Reviewer: ```suggestion
- **Postgres Dump & Restore**
```

Author: not what the screenshot says

> - **Upload Files** (CSV, Parquet, TXT)

Reviewer: ```suggestion
- **Upload Files**: CSV, Parquet, TXT
```

> - **Real-time Data Replication** (source connectors)

Reviewer: ```suggestion
- **Real-time Data Replication**: source connectors
```

> - **Migrations & Other Options**

Reviewer: ```suggestion
- **Migrations & other options**
```

Author: inconsistent with the rest of the bullets

  ![Data import in Tiger Cloud](https://assets.timescale.com/docs/images/tiger-cloud-console/data-import-wizard-in-tiger-cloud.png)

> A new data import component has been added to the overview dashboard, providing a clear view of your imports—including quick start, in-progress status, and completed imports:

Reviewer: ```suggestion
A new data import component has been added to the overview dashboard, providing a clear view of your imports. This includes quick start, in-progress status, and completed imports:
```
